templates/repo/issue/new_form.tmpl

```diff
@@ -129,28 +135,6 @@
 						</a>
 					{{end}}
 				</div>
-
-			<!-- input id="assignee_ids" name="assignee_ids" type="hidden" value="{{.assignee_id}}">
-			<div class="ui {{if not .Assignees}}disabled{{end}} floating jump select-assignee dropdown">
-				<span class="text">
-					<strong>{{.i18n.Tr "repo.issues.new.assignees"}}</strong>
-					<span class="octicon octicon-gear"></span>
-				</span>
-				<div class="filter menu">
-					<div class="no-select item">{{.i18n.Tr "repo.issues.new.clear_assignees"}}</div>
-					{{range .Assignees}}
-						<div class="item" data-id="{{.ID}}" data-href="{{$.RepoLink}}/issues?assignee={{.ID}}" data-avatar="{{.RelAvatarLink}}"><img src="{{.RelAvatarLink}}"> {{.Name}}</div>
-					{{end}}
-				</div>
-			</div>
-			<div class="ui select-assignee list">
-				<span class="no-select item {{if .Assignee}}hide{{end}}">{{.i18n.Tr "repo.issues.new.no_assignees"}}</span>
-				<div class="selected">
-					{{if .Assignee}}
-						<a class="item" href="{{.RepoLink}}/issues?assignee={{.Assignee.ID}}"><img class="ui avatar image" src="{{.Assignee.RelAvatarLink}}"> {{.Assignee.Name}}</a>
-					{{end}}
-				</div>
-			</div>-->
```

Comment:
Hello, I don't think these comments are usefull, so I suggest to remove them, If it have some special reason that should be kept. please tell me, I will recover them. thanks